demos/image_processing_demo/cpp/main.cpp

```diff
@@ -57,7 +57,8 @@ DEFINE_INPUT_FLAGS
 DEFINE_OUTPUT_FLAGS
 
 static const char help_message[] = "Print a usage message.";
-static const char at_message[] = "Required. Type of the model, either 'sr' for Super Resolution task, 'deblur' for "
+static const char at_message[] = "Required. Type of the model, either 'sr' for Super Resolution task, "
+                                 "'sr_channel_joint' for Super Resolution task that accepts and returns 1 channel image, 'deblur' for "
```

Comment:
`task` -> `model` ???

Reply:
Yes. Please change it. Also please update README Running section after that

Reply:
Done